### ISSUE TITLE
Clarifying MVP

### ISSUE THREAD

**Issue by arjunv27** [roomOnDemand/healthCareToHotels#5]

**Comment by arjunv27** [comment]
* Discover User Location
* Present User with places to stay near by

Let's keep it as simple as this for now

**Comment by arjunv27** [comment]
> Present User with places to stay

Requires aggregation of potential places to stay

**Comment by klunawat-realestate** [comment]
Agreed w/super simple. Makes sense to test with a few users next week

**Comment by arjunv27** [comment]
> test with a few users next week

Key for functionality is simplicity, this is moving fast though -- will reach out to some friends who can help scale.
Need hospitality data.

**Comment by klunawat-realestate** [comment]
Tell me more about what kind of hospitality data? Will work on it

**Comment by arjunv27** [comment]
Flow of the app is:

`Discover User Location -> 
     if (housing is available near location): Display Options for Housing 
     otherwise: collect User Email and location, message when options become available`

Need to discover availability and rates of available places, although there should be methods to subsidize the cost (as you mentioned previously crowdfunding)

**Comment by klunawat-realestate** [comment]
On it

**Comment by klunawat-realestate** [comment]
Are we also adding photos for each housing option?

**Comment by arjunv27** [comment]
Images would be useful

**Comment by klunawat-realestate** [comment]
Check tab Zeus in sheet here. It has 18 NY-based listings that are available as of today. Note: still awaiting discounted pricing from the company but this should help us get started:

https://docs.google.com/spreadsheets/d/1dl704K9ic3LqjAXkak1SGjLT7jEwXrjrUhiUP_6YB44/edit#gid=699497200

Will keep updating as we get more data

**Comment by klunawat-realestate** [comment]
Optional: speaking to Zeus management at 1.00 pm incase anyone wants to join
